gdal/ogr/ogrsf_frmts/dgn/ogrdgnlayer.cpp

```diff
@@ -74,6 +78,23 @@ OGRDGNLayer::OGRDGNLayer( const char * pszName, DGNHandle hDGNIn,
     }
     pszLinkFormat = CPLStrdup(pszLinkFormat);
 
+    const char * pszULinkType = const_cast<char *>(
+        CPLGetConfigOption( "DGN_ULINK_TYPE", "NONE" ) );
+
+    if( !EQUAL(pszULinkType,"NONE") )
+    {
+        char * testULink;
+        iULinkType = (int) strtol( pszULinkType, &testULink, 10 );
```

Comment:
The value of iULinkType seems to be compared only to -1. Shouldn't DGN_ULINK_TYPE be a boolean option tested with CPLTestBool ?

Reply:
and we need doc in the .rst pages of the driver(s) to document it

Reply:
actually do we still need that option ? Couldn't that be enabled systematically ?

Reply:
No, we don't need that option.